src/wing/CCPACSWing.h

```diff
@@ -64,125 +77,250 @@ friend class CTiglWingBuilder;
 
-    // Returns the mean aerodynamic chord of the wing
+    /**
+     * @brief Returns the mean aerodynamic chord of the wing
+     * @param mac_chord
+     * @param mac_x
+     * @param may_y
+     * @param mac_z
+     */
     TIGL_EXPORT void  GetWingMAC(double& mac_chord, double& mac_x, double& mac_y, double& mac_z) const;
 
-    // Calculates the segment coordinates from global (x,y,z) coordinates
-    // Returns the segment index of the according segment
-    // If x,y,z does not belong to any segment, -1 is returned
+    /**
+     * @brief Calculates the segment coordinates from global (x,y,z) coordinates
+     * If x,y,z does not belong to any segment, -1 is returned
+     * @param xyz Global (x,y,z) coordinates
+     * @param[in] eta
+     * @param[in] xsi
+     * @param[out] eta
+     * @param[out] xsi
+     * @param onTop
+     * @return int Returns -1 if x,y,z does not belong to any segment
+     */
     TIGL_EXPORT int GetSegmentEtaXsi(const gp_Pnt& xyz, double& eta, double& xsi, bool &onTop);
 
-    // Returns the Component Type TIGL_COMPONENT_WING.
+    /**
+     * @brief Returns the Component Type TIGL_COMPONENT_WING or TIGL_COMPONENT_ROTORBLADE.
+     */
     TIGL_EXPORT TiglGeometricComponentType GetComponentType() const override
     {
         return !IsRotorBlade() ? TIGL_COMPONENT_WING : TIGL_COMPONENT_ROTORBLADE;
     }
 
     TIGL_EXPORT TiglGeometricComponentIntent GetComponentIntent() const override {return TIGL_INTENT_PHYSICAL; }
 
-    // Returns the lower Surface of a Segment
+    /**
+     * @brief Returns the lower Surface of a Segment
+     * @param index Index of the segment
+     * @return Handle(Geom_Surface) Returns Handle to the lower surface of a segment
+     */
     TIGL_EXPORT Handle(Geom_Surface) GetLowerSegmentSurface(int index);
 
-    // Returns the upper Surface of a Segment
+    /**
+     * @brief Returns the upper Surface of a Segment
+     * @param index Index of the segment
+     * @return Handle(Geom_Surface) Returns Handle to the upper surface of a segment
+     */
     TIGL_EXPORT Handle(Geom_Surface) GetUpperSegmentSurface(int index);
 
-    // Get the guide curve segment (partial guide curve) with a given UID
+    /**
+     * @brief Get the guide curve segment (partial guide curve) with a given UID
+     * @param uid UID of the segment
+     * @return CCPACSGuideCurve&
+     */
     TIGL_EXPORT CCPACSGuideCurve& GetGuideCurveSegment(std::string uid);
 
-    // Returns all points that define the guide curves
+    /**
+     * @brief Returns all points that define the guide curves
+     * @return std::vector<gp_Pnt> Returns a vector containing all points definig the guide curves
+     */
     TIGL_EXPORT std::vector<gp_Pnt> GetGuideCurvePoints();
 
-    // Returns all guide curve wires as a compound
+    /**
+     * @brief Returns a wire consisting of all guide curves as a compound
+     * @return TopoDS_Compound Returns the OCC Type TopoDS_Compound
```

Comment:
See above. I personally find the forumulation "Returns the OCC Type TopoDS_Compound" a bit awkward.

Reply:
ok, @brief already explains what is being returned, I would leave the '@param <returntype>' comment without any further description